spec.md

```diff
@@ -217,26 +217,41 @@ Incompatible changes to the schema SHOULD be reflected by a different URL.
   or application protocol's content-type metadata property. Normative rules
   for the binary mode and the content-type metadata mapping can be found 
   in the respective transport mapping specifications.
-    
+  
 * Constraints:
   * OPTIONAL
   * If present, MUST adhere to the format specified in
     [RFC 2046](https://tools.ietf.org/html/rfc2046)
 * For Media Type examples see [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
 
 ### extensions
+
+### idendityLabels
```

Comment:
2 problems: 1 - there is a typo: `identity`.  2 - This name is too prescriptive to the problem you are trying to solve. You want to add labels and use them as identity, but at the top level there is only a subset of events that require identity. But all events could opt into labeling. 

just call it `labels`

Reply:
Will correct the typo. Thanks. Labels sounds too generic and the event producer can put anything inside this bag, which defeats the purpose of this attribute. This attribute is optional, so if an event producer does not have additional identity labels, then the event will not have this attribute field.

Reply:
This solves the special case of identity, but not the general case of labeling.